extended/src/main/java/io/kubernetes/client/extended/kubectl/rollout/Rollbacker.java

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package io.kubernetes.client.extended.kubectl.rollout;
+
+import io.kubernetes.client.common.KubernetesObject;
+
+public abstract class Rollbacker<ApiType extends KubernetesObject> implements RolloutAction {
```

Comment:
Calling it `Rollbacker` is a little weird since it really represents the complete rollout. Perhaps call it `Rollout` instead? (and change `StatefulSetRollbacker` etc)

Reply:
Agree , this makes more sense